google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -127,12 +127,14 @@ public HttpStorageRpc(StorageOptions options) {
     private final Storage storage;
     private final LinkedList<BatchRequest> batches;
     private int currentBatchSize;
+    private final LinkedList<RetryBatchCallBack> failedBatches;
 
```

Comment:
more accurate would be declare variables as List<...>, not as LinkedList<...>

Reply:
Done.